autosklearn/pipeline/components/base.py

```diff
@@ -340,10 +339,7 @@ def __init__(self, dataset_properties, random_state=None):
         # self.configuration = self.get_hyperparameter_search_space(
         #     dataset_properties).get_default_configuration()
 
-        if random_state is None:
```

Comment:
This is now done in the underlying model I guess?

Reply:
Likewise as above